Content.Server/Genetics/Components/GenomeComponent.cs

```diff
@@ -0,0 +1,36 @@
+using Content.Server.Genetics;
+using Content.Server.Genetics.Systems;
+using Robust.Shared.Prototypes;
+
+namespace Content.Server.Genetics.Components;
+
+/// <summary>
+/// Gives this entity a genome for traits to be passed on and potentially mutated.
+/// Both of those must be handled by other systems, on its own it has no functionality.
+/// </summary>
+[RegisterComponent, Access(typeof(GenomeSystem))]
+public sealed partial class GenomeComponent : Component
```

Comment:
i doubt anything will really need to use it in shared but i dont see any reason this + genes/genomeprototype/genomelayout/genome can't be in shared. gets rid of the registerignores as well. the system can stay in server probably